packages/codemod/src/utils.ts

```diff
@@ -326,3 +328,27 @@ export function replaceIdentifiers(
       path.value.name = "NextApiRequest"
     })
 }
+
+export const replaceBlitzPkgsVersions = async (
+  packageJson: {dependencies?: Record<string, any>},
+  npmTag: string,
+) => {
+  let blitzPkgVersion = npmTag
+  const result = await fetchDistTags("blitz")
+  if (npmTag in result) {
+    blitzPkgVersion = result[npmTag]
+  }
+
+  if (!packageJson.dependencies) {
+    packageJson.dependencies = {}
+  }
+
+  packageJson.dependencies["@blitzjs/next"] = blitzPkgVersion
+  packageJson.dependencies["@blitzjs/rpc"] = blitzPkgVersion
+  packageJson.dependencies["@blitzjs/auth"] = blitzPkgVersion
+  packageJson.dependencies["blitz"] = blitzPkgVersion
+  packageJson.dependencies["next"] = "12.2.0"
+  packageJson.dependencies["zod"] = "latest"
```

Comment:
I think we should add a hardcoded version for zod as well

Reply:
Let's fetch latest for zod? Otherwise, we'd make codemod depending on zod releases